README.rst

```diff
@@ -42,6 +42,14 @@ The built documentation will be available at::
 
 Any good internet browser will be able to display the documentation (see also `Serving the documentation`_).
 
+To build the documentation into a single page HTML document execute the following command::
+
+  make singlehtml
+
+The built documentation will be available at::
+
+  /absolute/path/to/cellml-specification/build/singlehtml/index.html
+
 To build the PDF form of the documentation simply execute the following command::
 
   make latexpdf
```

Comment:
Could we rename that rule to `pdf`? I imagine that `latexpdf` is the rule name used by Sphinx by default, but it really doesn't make sense to me. I just want a PDF, I don't care whether it first needs to go through LaTeX or not.

Reply:
I'd prefer to leave it at the default (I'm used to it being called latexpdf)